datasync/chaindatafetcher/kas/model.go

```diff
@@ -1,6 +1,9 @@
 package kas
 
-const TxTableName = "klay_transfers"
+const (
+	TxTableName          = "klay_transfers"
+	KctTransferTableName = "kct_transfers"
```

Comment:
Same comment like https://github.com/klaytn/klaytn/pull/587/files#r465504310

Reply:
Agreed, but the table name is used for the `gorm` to find the table.

Reply:
Understood. As we discussed, it is hard to change the table name as we need to change the read queries at the same time.